src/components/landing/Testimonials/SidebarTestimonial.js

```diff
@@ -33,21 +35,13 @@ const Testimonial = () => (
 
       <footer className="mt-8">
         <div className="flex items-start">
-          <div className="flex-shrink-0 inline-flex rounded-full border-2 border-white">
-
-          <picture>
-            <source srcSet={enriqueAvatar} type="image/webp" />
-            <source srcSet={enriqueAvatarPng} type="image/png" />
-
-            <img
-              src={enriqueAvatarPng}
-              alt="Enrique's face"
-              className="h-12 w-12 rounded-full"
-              height="100"
-              width="100"
-            />
-          </picture>
-          </div>
+          <Avatar
+            avatar={{
+              webp: enriqueAvatar,
+              png: enriqueAvatarPng,
+              alt: `The fact of a man with a slight beard and glasses. He is looking up and to the right slightly.`,
```

Comment:
```suggestion
              alt: `The face of a man with a slight beard and glasses. He is looking up and to the right slightly.`,
```

Too philosophical with 'fact'